### DISCUSSION TITLE
menu control

### DISCUSSION THREAD

**Discussion by fanlt** [luyadev/luya#2143] [General & Uncategorized]
I don't want menu bind with perspermissions ,So I want a menu control in the admin: can create,edit,delete menus!

By this User have perspermissions  with menu, but menu control can display or hidden this menu!

**Comment by nadar** [comment]
I don't understand the question. Please make more examples. The LUYA Admin Panel Menu can not be overwritten. 

ps: No need for `!` in the forums...

**Reply by fanlt** [reply]
Ok,Can any way to create,edit,delete menu in the admin panel?

**Comment by nadar** [comment]
https://luya.io/guide/app-admin-module-permission this section explains how to add new nodes and items. If you don't want to use LUYA Admin's [NgRest CRUD](https://luya.io/guide/ngrest-concept) you might use `itemRoute()` which can contain a Yii Framework typically module/controller/action routes.

Its currently not possible to have menu items without user permissions
